components/raftstore-v2/src/raft/storage.rs

```diff
@@ -183,6 +199,7 @@ impl<EK: KvEngine, ER: RaftEngine> Storage<EK, ER> {
             gen_snap_task: RefCell::new(Box::new(None)),
             split_init: None,
             apply_trace,
+            flushed_epoch: epoch,
```

Comment:
It seems this epoch is the newly epoch and irrelevant with `flush`.
Will it delete the file before ingest replayed after restart?

Reply:
Q1: `epoch` refresh only if `trace.admin.index` has writed in the disk.
Q2: sst will be delete if the sst's epoch is stale than the `flush epoch`